testkit/src/main/java/akka/kafka/testkit/internal/KafkaContainerCluster.java

```diff
@@ -101,24 +112,40 @@ public String getBootstrapServers() {
   }
 
   private Stream<GenericContainer> allContainers() {
-    Stream<GenericContainer> genericBrokers = this.brokers.stream().map(b -> (GenericContainer) b);
+    Stream<GenericContainer> genericBrokers = this.brokers.stream().map(b -> b);
```

Comment:
```suggestion
    Stream<GenericContainer> genericBrokers = this.brokers.stream();
```

Reply:
This and the other example you highlighted are actually required to cast the inner type to the desired type.  I'm not sure why an identity function is required, but it's essentially the same as:

```
Stream<GenericContainer> genericBrokers = this.brokers.stream().map(b -> (GenericContainer) b);
```

I made the cast explicit to avoid confusion.

Reply:
On second thought, the cast isn't actually necessary.  I've removed it.